cherab/openadas/parse/adf22.py

```diff
@@ -19,6 +19,8 @@
 from cherab.core.utility import RecursiveDict
 from .utility import parse_adas2x_rate
 
+CM3TOM3 = 1E-6
```

Comment:
same as for cherab/openadas/parse/adf22.py ?

Reply:
Done